### PR TITLE
Frontlight/Naturallight widgets: insert TitleBar, movable

### PR DESCRIPTION
**Pull request by hius07**
Removed the last old close button.
Both widgets are now movable, so removed jumping down/up of the Frontlight dialog when opening Naturallight configuration.

Before

After

I have no device with warmlight, so made it blindly, looking at the picture in https://github.com/koreader/koreader/pull/3744#issuecomment-372471223.
Need testing.

---
This change is [Reviewable](https://reviewable.io/reviews/koreader/koreader/8769)

### CODE REVIEW

**Comment by hius07** [comment]
Remove closebutton.lua?

**Comment by poire-z** [comment]
> Remove closebutton.lua?

Might still be used by user-plugins that were made from cut&pasting old stuff.
So, depends if we want them to force updating their code, or if we are good guys with crap in our garden that we can't throw away :/

**Review by poire-z — APPROVED** [review]
> Need testing.

Go ahead, let the users test it :)

**Comment by NiLuJe** [comment]
Unless I'm mistaken, this appears to have broken swiping inside the progress bar? (this just moves the whole widget now).

**Comment by hius07** [comment]
Maybe it was a bad idea to have fl widget movable, no need to see the text behind the widget while tuning the frontlight.
So the simpliest fix is to revert movability.

In the meantime we have movable Skim widget, swiping (actually panning) of the progress bar doesn't work there. Should we find the way to have these widgets movable and keep progress bar handling (horizontal) swipes?

**Comment by Frenzie** [comment]
If you want, but it's always seemed perfectly sensible behavior to me that they aren't.

**Comment by NiLuJe** [comment]
Not necessarily: I remember that we could simply make skimto transparent: I haven't checked if that changed, but that's better than a movable in my book.

TL;DR: Revert movable on FL widget, just make sure it can go transparent on hold (which it can't now, and may not have been able to before either).
